ci/azure-pipelines-windows.yml

```
@@ -0,0 +1,43 @@
+# Template for the basic Windows steps in Azure Pipelines
+
+steps:
+
+- bash: echo "##vso[task.prependpath]$CONDA\Scripts"
+  displayName: Add conda to PATH
+
+- bash: |
+    set -x -e
+    conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
```

```suggestion
    conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
    conda install pycryptosat --yes
    conda config --set sat_solver pycryptosat
    conda config --set safety_checks disabled
```

We're spending 8.5 min just installing dependencies. Try adding these lines to see if we can get conda to resolve the dependencies faster. Maybe it will cut down on the time a bit.